gcc/testsuite/jit.dg/test-name.c

```diff
@@ -0,0 +1,81 @@
+/* { dg-do compile { target x86_64-*-* } } */
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "libgccjit.h"
+
+// We want the debug info to check the function variable's name.
+#define TEST_ESCHEWS_SET_OPTIONS
+static void set_options (gcc_jit_context *ctxt, const char *argv0)
+{
+    gcc_jit_context_set_bool_option(ctxt, GCC_JIT_BOOL_OPTION_DEBUGINFO, 1);
```

Comment:
This already seems to be set by [the test harness](https://github.com/rust-lang/gcc/blob/0084a7397d3a3e540cf49c318d4292a0c704e841/gcc/testsuite/jit.dg/harness.h#L263).
Was there a reason to add it here?